site/content/docs/commands/job-package.en.md

```diff
@@ -16,20 +16,21 @@ $ copilot job package
   -n, --name string         Name of the job.
       --output-dir string   Optional. Writes the stack template and template configuration to a directory.
       --tag string          Optional. The container image tag.
+      --upload-resources    Optional. Whether to upload dependency resources (e.g., image that needs to build and push), so that the output of `copilot svc package` can be directly used for deployment.
```

Comment:
```suggestion
      --upload-resources    Optional. Whether to upload dependency resources (e.g., an image that needs to be built and pushed), so that the output of `copilot svc package` can be directly used for deployment.
```